web/gds-user-ui/src/components/BasicDetailsForm/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { VStack } from "@chakra-ui/react";
+import InputFormControl from "components/InputFormControl";
```

Comment:
@elysee15 great work. what do you think if we move InputFormControl  && SelectFormControl components on the UI folder?